### DISCUSSION TITLE
Erratic Left Thumbstick movement on controller

### DISCUSSION THREAD

**Discussion by Someguy1519** [mupen64plus-ae/mupen64plus-ae#950] [General]
On M64Plus FZ Pro version 3.0.291, the left thumbstick on my 8bitdo SN30 Pro+ seems to have a mind of its own when trying to map the thumbstick. It's registering random inputs that I'm not giving it. Seems to be affecting gameplay as well.

Device is a Galaxy S8+ running Android 9

**Comment by fzurita** [comment]
Try increasing the deadzone. When creating the profile, there is a menu on the top right corner to select the deadzone size.

**Reply by Someguy1519** [reply]
> Try increasing the deadzone. When creating the profile, there is a menu on the top right corner to select the deadzone size.

I tried that. Doesn't seem to have made any difference at all.

**Reply by fzurita** [reply]
Interesting, I'm releasing an update next week that will increase the default deadzone size since a lot of people are having this same problem. I'm surprised that increasing the deadzone is not doing anything for you though, it has fixed it for everyone else.

**Reply by Someguy1519** [reply]
> Interesting, I'm releasing an update next week that will increase the default deadzone size since a lot of people are having this same problem. I'm surprised that increasing the deadzone is not doing anything for you though, it has fixed it for everyone else.

I switched to a different controller profile I made for an Xbox One Controller a while back and it seems to be working fine now. Looks like the issue only occurs when making a new profile for the controller.
